IFC4x3/Properties/r/RailJointLength_2tL5QKZH14aRWa$kMNmQ64/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RailJointLength_2tL5QKZH14aRWakMNmQ64" Name="RailJointLength" UniqueId="b7545694-8d10-4491-b824-fee597c1a184" PrimaryDataType="IfcPositiveLengthMeasure" />
+
```

Comment:
Is this a nominal value or a quantity? Could "NominalLength" be used (context given by Pset)?

Reply:
Agree, I have used NominalLenght
Commit : b9da2b65d5470974f60d4ecfa44ab16135543152